### PR TITLE
docs: Update the doc after implementing online expansion

### PR DESCRIPTION
**Pull request by shuo-wu**
longhorn/longhorn#1674

### CODE REVIEW

File: content/docs/1.4.0/volumes-and-nodes/expansion.md [hunk 1]
```diff
@@ -109,3 +112,22 @@ If a volume is reverted to a snapshot with smaller size, the frontend of the vol
...
+3. Expand the filesystem.
+
+#### RWX volume
+
+Currently, Longhorn is unable to expand the filesystem (NFS) for RWX volumes. - If you decide to expand a RWX volume manually, you can:
+
+1. Expand the block device of the RWX volume via PVC or UI.
+2. Figure out the share manager pod of the RWX volume then execute the filesystem expansion command. The share manager pod is typically named as `share-manager-<volume name>`.
+    ```shell
+    kubectl -n longhorn-system exec -it <the share manager pod> -- resize2fs /dev/longhorn/<volume name>
```
Comment (innobead) [inline content/docs/1.4.0/volumes-and-nodes/expansion.md:132]: Need to add `xfs_growfs` for XFS.

File: content/docs/1.4.0/volumes-and-nodes/expansion.md [hunk 2]
```diff
@@ -3,23 +3,23 @@
   weight: 4
 ---
 
-Volumes are expanded in two stages. First, Longhorn expands the frontend (the block device), then it expands the filesystem.
+Volumes are expanded in two stages. First, Longhorn resizes the block device, then it expands the filesystem.
 
-To prevent the frontend expansion from interference by unexpected data R/W, Longhorn supports offline expansion only. The `detached` volume will be automatically attached to a random node with [maintenance mode.](../../concepts/#22-reverting-volumes-in-maintenance-mode)
-
-Rebuilding and adding replicas is not allowed during the expansion, and expansion is not allowed while replicas are rebuilding or being added.
+Since v1.4.0, Longhorn supports online expansion. Most of the time Longhorn can directly expand an attached volumes without limitations, no matter if the volume is being r/w or rebuilding.
```
Comment (innobead) [inline content/docs/1.4.0/volumes-and-nodes/expansion.md:8]: `.. no matter if the volume is being R/W or rebuilding` for consistency.

**Review by innobead — CHANGES_REQUESTED** [review]
LGTM, except minor comments.

File: content/docs/1.4.0/volumes-and-nodes/expansion.md (hunk 1 above)
Comment (shuo-wu) [reply content/docs/1.4.0/volumes-and-nodes/expansion.md:132]: Longhorn share manager pods are using `ext4`, should we add xfs?
https://github.com/longhorn/longhorn-share-manager/blob/e3ec44a91fa302f0280de6b93fddcd9525ef3bd8/app/cmd/server.go#L40

Comment (innobead) [reply content/docs/1.4.0/volumes-and-nodes/expansion.md:132]: My fault. You are right. The doc is correct!

**Review by innobead — APPROVED** [review]

**Pull request merged**